appendices/clustering.md

```diff
@@ -0,0 +1,35 @@
+# Clustering Details
+
+When a new event passes through the pipeline, the variants are identified via a proprietary Ragel FSM Based Tokenization process (80%-90% more efficient than regex) and stripped from the event, replaced with wildcards, and the remaining invariant components of the event are then compared to existing pattern sets to determine their similarities.
```

Comment:
nit: "event" might indicate something else. "log" is more appropriate here

Reply:
Changed to log.